gems/aws-sdk-core/lib/seahorse/client/h2/connection.rb

```diff
@@ -0,0 +1,196 @@
+if RUBY_VERSION >= '2.3'
+  require 'http/2'
```

Comment:
And if it's an earlier Ruby version? We need to have a user-friendly error when client creation is attempted in an older version of Ruby.

Reply:
Yep, sample: https://github.com/aws/aws-sdk-ruby/pull/1945/files#diff-02d74b79db5eac02ca88a5b6f2154efb
(the diff is hidden by Github :()

Basic, I avoid to throw build/load errors but errors out immediately when they are trying to initialize an AsyncClient